src/test/java/com/google/sps/TimingPropertiesTest.java

```diff
@@ -190,4 +190,32 @@ public void getNewQuestion_validParamters() {
             Assert.assertEquals(true, true);
         }
     }
+
+     @Test 
+    //Test if giveUserPoints is given null value(s) as a parameter
+    public void giveUserPoints_nullParameters() {
```

Comment:
this test never tests a null datastore. We should have one test for null entity and one for null datastore.